optimade/models/structures.py

```diff
@@ -146,7 +146,7 @@ class Species(BaseModel):
     @validator("chemical_symbols", each_item=True)
     def validate_chemical_symbols(cls, v):
         if not (v in EXTENDED_CHEMICAL_SYMBOLS):
-            raise ValueError(f"{v} MUST be in {EXTENDED_CHEMICAL_SYMBOLS}")
+            raise ValueError(f"{v} MUST be an element symbol, 'vacancy' or 'X'.")
```

Comment:
Isn't this a regression, since we're losing the dynamicity/variability? Maybe we could just write `an element symbol, e.g., "C", "He", or a special symbol: {EXTRA_SYMBOLS}`?

Reply:
Fair comment

```suggestion
            raise ValueError(f"{v} MUST be an element symbol, e.g., "C", "He", or a special symbol from {EXTRA_SYMBOLS}.")
```